console/src/components/Services/Data/TableBrowseRows/FilterQuery.js

```diff
@@ -171,9 +174,59 @@ const renderSorts = (orderBy, tableSchema, dispatch) => {
 class FilterQuery extends Component {
   componentDidMount() {
     const dispatch = this.props.dispatch;
+    let urlFilters = [];
+    if (typeof this.props.urlQuery.filter === 'string') {
+      urlFilters = [this.props.urlQuery.filter];
+    } else if (Array.isArray(this.props.urlQuery.filter)) {
+      urlFilters = this.props.urlQuery.filter;
+    }
+    this.props.curQuery.where = {
+      $and: urlFilters.map(filter => {
+        const parts = filter.split(';');
+        const col = parts[0];
+        const op = parts[1];
+        const value = parts[2];
+        return { [col]: { [op]: value } };
+      }),
+    };
+
+    let urlSorts = [];
+    if (typeof this.props.urlQuery.sort === 'string') {
+      urlSorts = [this.props.urlQuery.sort];
+    } else if (Array.isArray(this.props.urlQuery.sort)) {
+      urlSorts = this.props.urlQuery.sort;
+    }
+    this.props.curQuery.order_by = urlSorts.map(sort => {
+      const parts = sort.split(';');
+      const column = parts[0];
+      const type = parts[1];
+      const nulls = 'last';
+      return { column, type, nulls };
+    });
```

Comment:
Here, you are mutating the props, which is an anti-pattern in react. A component should not manage its props. `props` are literally  **the state that is managed by the component owner.** That's why they should be treated as immutable data. 

My proposition here would be to use local variables:
```js
    const where = {
      $and: urlFilters.map(filter => {
        const parts = filter.split(';');
        const col = parts[0];
        const op = parts[1];
        const value = parts[2];
        return { [col]: { [op]: value } };
      }),
    };

    const order_by = urlSorts.map(sort => {
      const parts = sort.split(';');
      const column = parts[0];
      const type = parts[1];
      const nulls = 'last';
      return { column, type, nulls };
    });
```

And then use them in `setDefaultQuery` function:
```js
dispatch(setDefaultQuery({ where, order_by }));
```